packages/suite/src/hooks/wallet/useRbfForm.ts

```diff
@@ -120,7 +120,7 @@ const getRbfFeeInfo = (info: FeeInfo, rbfParams: RbfTransactionParams) => {
 const useRbfState = ({ selectedAccount, rbfParams, chainedTxs }: UseRbfProps) => {
     const { account, network } = selectedAccount;
 
-    const symbolFees = useSelector(state => state.wallet.fees[account.symbol]);
+    const symbolFees = useSelector(state => selectNetworkFeeInfo(state, account.symbol));
```

Comment:
nit, you can ignore:
```suggestion
    const networkFees = useSelector(state => selectNetworkFeeInfo(state, account.symbol));
```

Reply:
822ac511